ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly.yaml

```diff
@@ -1963,6 +1952,18 @@ tests:
     test:
     - chain: openshift-e2e-test-qe
     workflow: cucushift-installer-rehearse-vsphere-ipi-sdn
+- as: vsphere-ipi-sdn-migration-ovn-p2-f7
```

Comment:
Also need to remove `vsphere-upi-ovn-sdn2ovn`, right?

Reply:
yes,  we have `vsphere-ipi-sdn-migration-ovn-p2-f7` can cover vsphere

Reply:
To get those unneeded jobs removed in this PR to take effect, we can let this PR go, and use a follow up PR to remove `vsphere-upi-ovn-sdn2ovn` if that's also unneeded.